statuspage/js/s3.js

```diff
@@ -70,7 +70,7 @@ checkup.storage = (function() {
 				doneCallback(checksLoaded);
 			} else {
 				for (var i = 0; i < list.length; i++) {
-					let url;
+					var url;
```

Comment:
Oh, right, Safari doesn't support ES6... why do people still use Safari again? 😛 

Anyway this is fine, then, we can stick with ES5 for now.